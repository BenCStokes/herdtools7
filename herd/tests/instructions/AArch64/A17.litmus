AArch64 A17

(* Tests strh (register), SXTW*)
{ int16_t 0:X0 = 44; 0:X1=x; uint64_t x; 0:X2=0; }

P0;
  STRB W0, [X1, X2, SXTW];

exists (x=44)
