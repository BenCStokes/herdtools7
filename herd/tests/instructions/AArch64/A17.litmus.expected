Test A17 Allowed
States 1
[x]=44;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists ([x]=44)
Observation A17 Always 1 0
Hash=e59d8739bd436c143a2025bcfe9b9d18

